### DISCUSSION TITLE
Vídeo 10 - Desenvolvimento

### DISCUSSION THREAD

**Discussion by thpoiani** [thpoiani/lab-quarkus#13] [General]
Esse é o último vídeo de desenvolvimento.
Dessa vez deixamos um endpoint exposto para exibir os resultados para a abordagem de **Server Sent Events**.

Como você desenvolveria um client (frontend) para consumir esse endpoint e exibir o resultado da eleição?

**Comment by dbengp** [comment]
Aviso de Migração de Dependência do Quarkus

Prezado thpoiani, 

Usando o **Quarkus na verção 3.25.3**, foi identificado um problema de resolução da dependência **quarkus-resteasy-reactive**. Após investigação, descobri que essa dependência foi realocada no repositório central do Maven: [io.quarkus » quarkus-rest » 3.15.6](https://mvnrepository.com/artifact/io.quarkus/quarkus-resteasy-reactive)

Quem for usar versões mais recentes do Quarkus, substitua a dependência quarkus-resteasy-reactive pela nova dependência quarkus-rest.
```
Antes:

<dependency>
  <groupId>io.quarkus</groupId>
  <artifactId>quarkus-resteasy-reactive</artifactId>
</dependency>

Depois:

<dependency>
  <groupId>io.quarkus</groupId>
  <artifactId>quarkus-rest</artifactId>
</dependency>
```
#### seu projeto é muito elucidativo!

**Reply by thpoiani** [reply]
Obrigado pelo aviso! 🙏

De fato, a dependência `quarkus-resteasy-reactive` foi descontinuada e substituída por `quarkus-rest` nas versões mais recentes do Quarkus.  

https://quarkus.io/guides/rest-migration

Abri a issue https://github.com/thpoiani/lab-quarkus/issues/16 para rastrear essa atualização no projeto.
